[2025-01-22T11:05:05.658+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-22T11:05:05.669+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: branching_variables.branch manual__2025-01-22T11:05:01.388569+00:00 [queued]>
[2025-01-22T11:05:05.673+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: branching_variables.branch manual__2025-01-22T11:05:01.388569+00:00 [queued]>
[2025-01-22T11:05:05.674+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-22T11:05:05.683+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): branch> on 2025-01-22 11:05:01.388569+00:00
[2025-01-22T11:05:05.688+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2362) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-22T11:05:05.690+0000] {standard_task_runner.py:72} INFO - Started process 2364 to run task
[2025-01-22T11:05:05.689+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'branching_variables', 'branch', 'manual__2025-01-22T11:05:01.388569+00:00', '--job-id', '259', '--raw', '--subdir', 'DAGS_FOLDER/branching_variables.py', '--cfg-path', '/tmp/tmpd3ibek_u']
[2025-01-22T11:05:05.692+0000] {standard_task_runner.py:105} INFO - Job 259: Subtask branch
[2025-01-22T11:05:05.720+0000] {task_command.py:467} INFO - Running <TaskInstance: branching_variables.branch manual__2025-01-22T11:05:01.388569+00:00 [running]> on host 7854821b9569
[2025-01-22T11:05:05.874+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohamed' AIRFLOW_CTX_DAG_ID='branching_variables' AIRFLOW_CTX_TASK_ID='branch' AIRFLOW_CTX_EXECUTION_DATE='2025-01-22T11:05:01.388569+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-22T11:05:01.388569+00:00'
[2025-01-22T11:05:05.875+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-22T11:05:05.882+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-22T11:05:05.887+0000] {python.py:240} INFO - Done. Returned value was: filter_southest
[2025-01-22T11:05:05.888+0000] {branch.py:38} INFO - Branch into filter_southest
[2025-01-22T11:05:05.890+0000] {skipmixin.py:230} INFO - Following branch ('filter_southest',)
[2025-01-22T11:05:05.895+0000] {skipmixin.py:278} INFO - Skipping tasks [('filter_southwest', -1)]
[2025-01-22T11:05:05.916+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-22T11:05:05.917+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=branching_variables, task_id=branch, run_id=manual__2025-01-22T11:05:01.388569+00:00, execution_date=20250122T110501, start_date=20250122T110505, end_date=20250122T110505
[2025-01-22T11:05:05.945+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-22T11:05:05.962+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-22T11:05:05.966+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
